eng/pipelines/tasks/APIScan.yml

```diff
@@ -0,0 +1,33 @@
+steps:
+- task: DownloadPipelineArtifact@2
+  displayName: Download Pipeline Artifact
+  inputs:
+    source: "current"
+    path: "$(Pipeline.Workspace)"
+
+- task: CopyFiles@2
+  displayName: 'Copy Files to: $(Pipeline.Workspace)\ApiScanFiles'
+  inputs:
+    SourceFolder: '$(Pipeline.Workspace)\Lab.Release'
+    Contents: |
+      **\*Microsoft@(*.dll|*.pdb|*.exe)
+      **\*Newtonsoft@(*.dll|*.pdb|*.exe)
+      **\*OpenDebugAD7@(*.dll|*.pdb|*.exe)
+      **\*WindowsDebugLauncher@(*.dll|*.pdb|*.exe)
+      !**\*.resources.dll
+    TargetFolder: '$(Pipeline.Workspace)\ApiScanFiles'
+    CleanTargetFolder: true
+    OverWrite: true
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-apiscan.APIScan@2
+  displayName: 'Run APIScan'
+  inputs:
+    softwareFolder: '$(Pipeline.Workspace)\ApiScanFiles'
+    softwareName: MIEngine
+    softwareVersionNum: '$(Build.BuildNumber)'
+    symbolsFolder: 'SRV*http://symweb;$(Pipeline.Workspace)\ApiScanFiles'
+    isLargeApp: false
+    verbosityLevel: none
```

Comment:
Do we have to define `verbosityLevel`? Tasks usually dont require it and will override a more verbose setting if we run with `diagnostics logging` enabled.